grammar.js

```diff
@@ -48,7 +55,10 @@ module.exports = grammar({
     [$._expression, $.formal_parameters],
     [$.labeled_statement, $._property_name],
     [$.assignment_pattern, $.assignment_expression],
-    [$.computed_property_name, $.array]
+    [$.computed_property_name, $.array],
+    [$.jsx_identifier, $.jsx_attribute],
+    [$.jsx_identifier, $.jsx_namespace_name],
+    [$.jsx_identifier, $.jsx_member_expression]
```

Comment:
Is there a way to make these fixes without introducing these conflicts? There might not be, but since each conflict carries some runtime cost, it'd be good to double check that we're modeling things in the most efficient way. Besides the naming changes, what new constructs are we parsing that we couldn't before?

Reply:
Ah, reading this a bit more, I can see that we're parsing JSX fragment syntax now. It'd be nice to have a test case that's specifically about that.